### DISCUSSION TITLE
Vercel instructing me to remove conflicting A records

### DISCUSSION THREAD

**Discussion by coryswainston** [vercel/community#4993] [Help] [answered]
### Summary

I'm trying to point mycustomdomain.com and www.mycustomdomain.com to my Vercel web app.

www.mycustomdomain.com is working fine, but I am getting a message of "Invalid configuration" for mycustomdomain.com

It says "Please remove the following conflicting DNS record from your DNS provider." Then it lists two A records with IP addresses. Then it tells me to add the Vercel record (76.76.21.21). 

Looking at my DNS settings, I only see two A records both with name '@'. One is the Vercel one (76.76.21.21) and the other just says "Parked". 

What do I need to do to fix this? Any help appreciated greatly

### Steps to Reproduce

N/A or see summary

**Comment by amyegan** [comment]
Hey Cory. If you let me know the actual domain, I can check the records for you. Otherwise, this troubleshooting guide has some good things to check: https://vercel.com/docs/projects/domains/troubleshooting

**Comment by Barefoot-Dev** [comment]
I have the exact same situation, my domain is https://aliveai.xyz. Could you please check it @amyegan?

**Reply by amyegan** [reply]
It looks like that domain has some extra DNS records. Probably leftover from a previous website if you were using the domain elsewhere in the past. Since you're adding it to a Vercel project, you should keep `76.76.21.21` and delete the other two.

[whatsmydns.net/#A/aliveai.xyz](https://www.whatsmydns.net/#A/aliveai.xyz)

**Reply by Barefoot-Dev** [reply]
Hmm okay thanks - I can't actually see where these others are configured (in GoDaddy) - I'll reach out to them.

**Reply by adherb** [reply]
@Barefoot-Dev did you solve this? I'm having the same issue with aa GoDaddy domain.

**Reply by dostontech** [reply]
me too with godaddy i go same issues

**Comment by coryswainston** [comment, marked as answer]
For me the solution was to remove the "Parked" record.

**Reply by aanifowose111** [reply]
Where is the parked domain living on Godaddy?

**Reply by acarmagnani** [reply]
That worked for me too. Thanks.

**Reply by adherb** [reply]
Worked for me also!

**Reply by Syeda-Zahra79** [reply]
Worked for me too!!

**Reply by fahidsarker** [reply]
Worked for me as well. Thanks

**Reply by andrewcastle10** [reply]
+1, worked for me as well. Great answer

**Comment by yamas-developers** [comment]
I am seeing this "Remove the following conflicting DNS record from your DNS provider" every time and even though I have checked and there is no such record exists there in hpanel. the www.mydomain.net is added successfully and working fine but mydomain.net is showing below error
<img width="928" alt="Screenshot 2024-04-03 at 3 28 43 PM" src="https://github.com/vercel/community/assets/142791023/beff0240-b34c-4bcb-a345-92c877810760">

**Reply by heysolomon** [reply]
having the same issue

**Reply by amyegan** [reply]
Please remove the conflicting DNS record. Then the configuration message will go away. 🙏 

If you share the domain, someone here may be able to help you check the records. You can also find tips for DIY troubleshooting domains and DNS issues at https://vercel.com/docs/projects/domains/troubleshooting

**Reply by Rostom2** [reply]
Hello i have the same problem here : 
I deleted the old configuration but in goDaddy we cannot delete the configurations it tells me about
![Vercel2](https://github.com/vercel/community/assets/34664987/49242e64-ec7d-4084-9c51-70610d3d883b)

**Reply by TylerBuese** [reply]
Hello,

I'm also having the same problem - any tips? These A records don't exist.

<img width="729" alt="image" src="https://github.com/vercel/community/assets/36856818/fa6b6919-3015-4d8f-9f05-4026ec0207e7">

**Reply by amyegan** [reply]
Please reach out to your registrar if you're still unable to find the records to remove. They have access to your domain info and can help you remove the conflicting records.
